### DISCUSSION TITLE
For s3 bucket "s3-public-policy" policy getting 15 notification emails with same output.

### DISCUSSION THREAD

**Discussion by nitinitare** [cloud-custodian/cloud-custodian#7327] [General]
Hi All,

I am running a policy to get notification mail with the list of s3 buckets which have public access.

I am running the below command :
 **custodian run -s out s3publicaccess.yml --region all**

If i am specifying one region not getting the list of all buckets.

Please suggest what should be done in this case.

**Comment by kapilt** [comment]
s3 buckets appear in all regions, so access policies on s3 should ideally pin to a single region (either via policy region attribute or executing against one region). alternatively they can use a bucket location filter to target buckets in particular regions.

**Comment by nitinitare** [comment]
@kapilt Can you please provide me more info on the bucket location filter, didn't get anything related to it in documentation.
